pkg/process/v6/transformers/entry.go

```diff
@@ -9,21 +9,27 @@ import (
 )
 
 type RelatedEntries struct {
-	VulnerabilityHandle grypeDB.VulnerabilityHandle
+	VulnerabilityHandle *grypeDB.VulnerabilityHandle
```

Comment:
This became a pointer because KEV records don't have a vulnerability handle, so it needs to become optional? Just checking my understanding here.

Reply:
thats right -- where the provider objects previously was always on the VulnerabilityHandle, we still need this object but there is no VulnerabilityHandle with KEV records (since they decorate already existing vuln records).